permissions/plugin-extra-tool-installers.yml

```
@@ -2,11 +2,13 @@
 name: "extra-tool-installers"
 github: "jenkinsci/extra-tool-installers-plugin"
```

```suggestion
github: &gh "jenkinsci/extra-tool-installers-plugin"
```
The reference is needed for your issue tracker change below.